historypanewidget.cc

```diff
@@ -137,7 +138,7 @@ void HistoryPaneWidget::deleteSelectedItems()
 
   // Need to sort indexes in the decreasing order so that
   // the first deletions won't affect the indexes for subsequent deletions.
-  qSort( idxsToDelete.begin(), idxsToDelete.end(), qGreater<int>() );
+  std::sort( idxsToDelete.begin(), idxsToDelete.end() );
```

Comment:
The behavior change is here. `#include <functional>` and use `std::greater<int>`.